x-pack/plugin/core/template-resources/src/main/resources/traces@mappings.json

```diff
@@ -0,0 +1,28 @@
+{
+  "template": {
+    "mappings": {
+      "date_detection": false,
+      "properties": {
+        "@timestamp": {
+          "type": "date"
+        },
+        "data_stream.type": {
+          "type": "constant_keyword",
+          "value": "traces"
```

Comment:
Are we ok with this here or should we move that to `traces-otel@mappings`? Related: https://github.com/elastic/elasticsearch/pull/111091#discussion_r1705736505

APM uses the same value, doesn't it?

Reply:
Yes, it's fine to have it here.

Reply:
We should also delete the `traces@mappings` in APM data and rely on this one instead.

Reply:
> We should also delete the `traces@mappings` in APM data and rely on this one instead.

Yeah, that was my plan. 

I did that here: https://github.com/elastic/elasticsearch/pull/111091/commits/f42756445d4e409b12d23a26ed85fa3b7a6508a0#diff-6a6ef4571ed7574fdc81c6f5bdaadce140dc08c70ff2c947394f71b604d2169b

Did I overlook something?

Reply:
Sorry, I overlooked that.